backend/src/graphql/resolver/TransactionLinkResolver.ts

```diff
@@ -258,7 +258,7 @@ export class TransactionLinkResolver {
           }
         }
 
-        const creations = await getUserCreation(user.id, false)
+        const creations = await getUserCreation(user.id)
```

Comment:
why is the field exported in singular when the function definition itself is plural (trailing s). This seems inconsistent